### DISCUSSION TITLE
Config file placement

### DISCUSSION THREAD

**Discussion by jarjk** [kdheepak/taskwarrior-tui#540] [Q&A] [answered]
Is `taskwarrior`'s `taskrc` configuration file read, if is not located under `$HOME/.taskrc` but rather `$HOME/.config/task/taskrc` because I'm afraid, it's ignored.

**Comment by kdheepak** [comment]
The `taskwarrior-tui` should do exactly the same thing as what the `task` binary would do. Is that not happening for you?

**Comment by jarjk** [comment]
I'm afraid not. This is how my config file, under `$HOME/.config/task/taskrc` ends
```
# taskwarrior-tui
uda.taskwarrior-tui.report.next.filter=status:pending 
```
but when I open up `taskwarrior-tui`, the basic filter is applied: `status:pending -WAITING limit:page`.

**Reply by kdheepak** [reply]
I wonder why. Taskwarrior TUI calls task as a subprocess.

**Reply by jarjk** [reply]
yeah, I dunno either
Could you try to reproduce the issue, or should I try other config options to see whether they work?

**Reply by jarjk** [reply, marked as answer]
I added the following line, which did the trick

`report.next.filter=status:pending`
